code/game/gamemodes/miniantags/abduction/machinery/experiment.dm

```diff
@@ -216,6 +216,36 @@
 	occupant = null
 	update_icon(UPDATE_ICON_STATE)
 
+/obj/machinery/abductor/experiment/relaymove()
+	if(!occupant)
+		return
+	to_chat(occupant, "<span class='warning'>You start mashing alien buttons at random!</span>")
```

Comment:
In my opinion, this message could be a bit more descriptive for the player about what is happening. Instead of mashing alien buttons, maybe have it say "You start trying to break free!" or something similar.